package.json

```diff
@@ -33,4 +33,4 @@
   "workspaces": [
     "packages/*"
   ]
-}
+}
```

Comment:
might want to change your settings to ensure EOL before EOF

Reply:
.json files dont have EOL before EOF. Only js files do. I will add them if you want to.